helm/superset/templates/deployment.yaml

```diff
@@ -114,6 +114,24 @@ spec:
             - name: http
               containerPort: {{ .Values.service.port }}
               protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: {{ .Values.service.port }}
```

Comment:
Just use the name of the port here (`http`)

Reply:
Thanks for you comment, i changed these ;)